src/Hit/Issue.hs

```diff
@@ -34,7 +35,12 @@ runIssue issue me = case issue of
 getAllIssues :: Maybe Text -> IO ()
 getAllIssues me = withOwnerRepo (\t o r -> issuesForRepo' t o r stateOpen) >>= \case
     Left err -> errorMessage $ show err
-    Right is -> for_ (my is) (putTextLn . showIssueName blueCode)
+    Right is -> do
+      let maxLen = Fmt.maxLenOn (show . issueNumber) is
+      for_ (my is) $ \i -> do
+        let thisLen = T.length $ show (issueNumber i)
+            padSize = maxLen - thisLen
```

Comment:
I'm a bit confused about why you need to subtract `thisLen` from `maxLen` :thinking: According to the code of `padRight` already does this so it should be enough to pass `maxLen`

Reply:
Because I didn't use padRight. Please see the comment below.